.github/workflows/build_rocks.yaml

```diff
@@ -224,3 +226,23 @@ jobs:
         env:
           TRIVY_USERNAME: ${{ github.actor }}
           TRIVY_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+      - name: Check trivyignore
```

Comment:
Do you think we should write this in nodeJS given that we want to start going down that path throughout this cycle?

Reply:
I prefer to land this as is if you're fine with it - I think we need some initial setup around rewriting in javascript and also some research. The story itself started before we decided on the scope of rewriting ops workflows in javascript.